invite/test/invitation_record.test.ts

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  */
 
+import {dbConnect, Database} from '../src/db';
+import {setupDb} from '../src/setup_db';
 import {Invite, InviteAction} from '../src/types';
 import {InvitationRecord} from '../src/invitation_record';
 
+jest.mock('../src/db', () => require('./__mocks__/db'));
```

Comment:
Don't mock, you can inline the in-memory db initialization in the `beforeAll()` and make the behaviour more transparent

Reply:
a) This is literally exactly what you did in bundle-size
b) This mock will be useful for e2e tests as well
c) Mocking is not evil

Reply:
It's not evil, but it can be confusing and adds; I did that in bundle-size because the implementation was much different at first, and when I changed how the implementation works I guess I never followed up with a refactor. The reason I did the refactor though, was that I _couldn't_ write the test without a mock - which is a sign of tight-coupling

Reply:
Done